### DISCUSSION TITLE
Help Wanted - Feature - Custom Error Pages

### DISCUSSION THREAD

**Discussion by taanguyen** [ashutoshkrris/EazyLoader#41] [Q&A] [answered]
hi @ashutoshkrris! I'm running into a Bad Credential Exception-- Wrong Password. Isn't the password arbitrary?
![Screen Shot 2021-10-27 at 10 16 56 PM](https://user-images.githubusercontent.com/13709437/139191566-084eb49a-efa6-4e6c-94ed-da98db338dfe.png)

**Comment by ashutoshkrris** [comment, marked as answer]
Please comment out the `self.loader.login()` line in `core/utils/ig_downloader.py` during the development. 
Before commiting, make sure you remove the comment.

**Reply by taanguyen** [reply]
thank you!
